src/std/main.ab

```diff
@@ -40,6 +40,13 @@ pub fun lines(text: Text): [Text] {
     return result
 }
 
+pub fun words(text: Text): [Text] {
+  let result = [Text]
+  let trimmed = trim(text)
+  unsafe $IFS=\$' ' read -rd '' -a {nameof result} <<<"\${nameof trimmed}"$
```

Comment:
```suggestion
  unsafe $read -rd '' -a {nameof result} <<<"\${nameof text}"$
```

This seems to fix everything @kilianc without use of any external functions. What do you think?